### DISCUSSION TITLE
Documentation lookup

### DISCUSSION THREAD

**Discussion by nathany** [zed-industries/zed#17723] [Feature Requests]
When hovering over functions and keywords in Rust code, the documentation appears. 

I'm wondering if there is any way to search this documentation within Zed? I'm thinking of something similar to Dash or Zeal but built into the editor.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
